android/src/main/java/com/pdftron/pdftronflutter/helpers/PluginUtils.java

```diff
@@ -2898,6 +2909,47 @@ private static void importAnnotations(String xfdf, MethodChannel.Result result,
         }
     }
 
+    private static void mergeAnnotations(String xfdf, MethodChannel.Result result, ViewerComponent component) throws PDFNetException {
+        PDFViewCtrl pdfViewCtrl = component.getPdfViewCtrl();
```

Comment:
This duplicates the logic in `importAnnotations`, could you rework that to include this case?